From f8ac48abded02353918b02db38629988bd111c46 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 27 Jun 2017 08:19:44 -0700
Subject: [PATCH 1/2] Include stdlib.h for exit() API
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes
main.c:81: warning: incompatible implicit declaration of built-in function ‘exit’

Patch from Debian
http://sources.debian.net/src/gtkperf/0.40%2Bds-2/debian/patches/01-include_stdlib.patch/

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 src/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/main.c b/src/main.c
index 888eb36..c226799 100644
--- a/src/main.c
+++ b/src/main.c
@@ -11,6 +11,7 @@
 #endif
 
 #include <getopt.h>
+#include <stdlib.h>
 
 #include "interface.h"
 #include "support.h"
-- 
2.13.2

